torch/export/_unlift.py

```diff
@@ -298,10 +294,20 @@ def _create_stateful_graph_module(
         delattr(submod, field)
         _assign_attr(buffer, stateful_gm, constant_fqn, attr_kind=_AttrKind.CONSTANT)
 
+    # Constants are not preserved well when we create a new GraphModule unlike param/buffers
+    for const_name, value in ep.constants.items():
+        if isinstance(value, torch.Tensor):
```

Comment:
should we have a check that this attribute is None, to make it clear this is only needed for aliased constants?

Reply:
Hmm you mean attribute is set None? Yeah i can.